README.md

```diff
@@ -11,6 +13,25 @@ To generate the HTML files from markdown in the `/source` directory:
 3. `pip install recommonmark`
 4. Type `make html`
 
-To contribute to the documentation please fork this repository and create a pull request. If you have not done so already, please complete the [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/). 
+## Contributing
+
+**How to create Pull Requests to edit or create Mattermost documentation (by community members):**
+
+1. Sign the Contributor License Agreement (see instructions in the next section).
+3. On the Mattermost Documentation page that you want to edit, click the GitHub icon on the upper right corner that says "Edit".
+4. Click "Edit this file" (pencil icon).
+5. After making changes, check the "Create a new branch for this commit and start a pull request".
+6. Make sure that the Pull Request has a descriptive title. Add comments to briefly tell others what you have worked on (optional).
+7. Click "Create a Pull Request".
+
+**Signing CLA:**
+
+Please read the [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/) and sign it (at the bottom of the page), so you can be added to the Mattermost [Approved Contributor List](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true).
+
+**Additional process for Core Committers for Doc Repo:**
```

Comment:
Sorry, I should have provided some context for the additions I made when I created the PR - I have added the information in the ReadMe as per my discussion with Ian, and we will eventually move it to its own document. So the location of the additional section itself is fine for now, since this is temporary.